charts/telepresence/templates/deployment.yaml

```diff
@@ -41,14 +41,23 @@ spec:
             value: "443"
           - name: CLUSTER_ID
             value: {{ .Values.clusterID }}
+          {{- if (and .Values.agentInjector.create .Values.agentInjector.agentImage.registry) }}
+          - name: TELEPRESENCE_REGISTRY
+            value: {{ .Values.agentInjector.agentImage.registry }}
+          {{- else }}
           - name: TELEPRESENCE_REGISTRY
```

Comment:
I think we can / should remove this value so that we are only setting the environment variable once and that it's coming from the `.Values.agentInjector.agentImage.registry` value only.